app/views/groups/index.html.erb

```diff
@@ -1,5 +1,4 @@
 <section class="h-full w-full bg-white min-h-screen flex flex-col gap-4">
```

Comment:
- Make sure the Group page is as the given design

## Your design
![Screenshot from 2022-11-20 00-17-05](https://user-images.githubusercontent.com/80149314/202873970-29399ec7-02ef-4135-aeda-c5db45cd566c.png)

- It's required for you to use the exact design for your app. The colors, typography must be the same as the in the [mockup](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=)

Reply:
Dear Reviewer, 
To build and run with styling working, kindly use `bin/dev` instead of `rails s`. This is becuase i used tailwindcss for styling. 
Sincerely 
Elias

Reply:
Well done!

Reply:
@DeliceLydia , Thank you